### PR TITLE
Improved tsuru.yaml.rst

### PR DESCRIPTION
**Pull request by Abubakrce19**
Fixed many Grammatical issues.
Thank you!!

### CODE REVIEW

**Review by tcarreira — APPROVED** [review]

**Comment by Abubakrce19** [comment]
@tcarreira 
Hi thank you for the merge.
Am I counted for the swags ??

**Comment by tcarreira** [comment]
> Am I counted for the swags ??
This pr should be automatically counted. Let me know if it was not ;)

**Comment by Abubakrce19** [comment]
My 2 prs was meged one by tsuru and one by clapper but only 1 is counted why
Thank you 💞
@tcarreira 
Screenshot_2022-10-25-10-56-22-46

File: docs/using/tsuru.yaml.rst [hunk 1]
```diff
@@ -10,9 +10,9 @@ tsuru.yaml
 tsuru.yaml is a special file located in the root of the application. The name of
 the file may be ``tsuru.yaml`` or ``tsuru.yml``.
 
-This file is used to describe certain aspects of your app. Currently it describes
+This file is used to describe certain aspects of your app. Currently, it describes
 information about deployment hooks and deployment time health checks. How to use
-this features is described below.
+these features is described below.
```
Comment (arthurcgc) [inline docs/using/tsuru.yaml.rst:15]: there are still grammar errors in here... it should be "these features ARE described bellow"

File: docs/using/tsuru.yaml.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ tsuru provides some deployment hooks, like ``restart:before``, ``restart:after``
 and ``build``. Deployment hooks allow developers to run commands before and after
 some commands.
 
-Here is an example about how to declare this hooks in your tsuru.yaml file:
+Here is an example about how to declare these hooks in your tsuru.yaml file:
```
Comment (arthurcgc) [inline docs/using/tsuru.yaml.rst:27]: Here is an example about how to declare these hooks in your tsuru.yaml file: > "Here is an example ON how to declare these hooks in your tsuru.yaml file:"

**Pull request merged**